src/Containers/LeftSideBar/LeftSideBar.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { TextFields } from '@styled-icons/material/TextFields';
+import { ImageAlt } from '@styled-icons/boxicons-solid/ImageAlt';
+import { Layout3 } from '@styled-icons/remix-fill/Layout3';
+import { ListNumbered } from '@styled-icons/icomoon/ListNumbered';
+import { Dollar } from '@styled-icons/boxicons-regular/Dollar';
+import { Qrcode } from '@styled-icons/icomoon/Qrcode';
+import { Settings } from '@styled-icons/ionicons-sharp/Settings';
+import { 
+    DragDropContext, 
+    Droppable,
+    DroppableProvided,
+    DroppableStateSnapshot
+} from 'react-beautiful-dnd';
+import { ReceiptElements, draggableComponentsObj } from './ReceiptElements';
+import { CollapsibleHeader } from './CollapsibleHeader';
+import { List } from '../List';
+import { SearchBar } from '../../Inputs/SearchBar';
+import { MainInterface, ResponsiveInterface } from '../../Utils/BaseStyles';
+
+const isToggleable = true;
+const isLeftToggle = true;
+const dropDisabled = true;
+const backgroundColor = '#f5f5f5';
```

Comment:
greyBackgroundColor is more clear but both work. Note that it should be all caps if a true constant
Ex: BACKGROUND_COLOR